source/docs/software/advanced-gradlerio/gradlew-tasks.rst

```diff
@@ -7,14 +7,15 @@ Build tasks
 -----------
 
 ``./gradlew build`` - Assembles and tests this project. Useful for prebuilding your project without deploying to the roboRIO.
+
 ``./gradlew clean`` - Deletes the build directory.
 
 CompileCommands tasks
 ---------------------
 
-``./gradlew generateCompileCommands`` - Generate compile_commands.json. This is a configuration file that is supported by many Integrated Development Environments.
+``./gradlew generateCompileCommands`` - Generate compile_commands.json for C++ programs. This is a configuration file that is supported by many Integrated Development Environments.
```

Comment:
compile_commands.json is supported by the clangd language server maybe that should be mentioned here?

Reply:
I've seen as many reports of problems with clangd language server as I have successes, so I'm not sure it makes sense to call attention to it specifically

Reply:
I think it might be worth it for the sake of accuracy as it isn't the IDE which supports the compile_commands.json file but instead the Clangd language server. If we don't want to specifically mention Clangd it might be worth it to just say something like: "supported by the language server run in many Integrated Developement Environments" or something along those lines?